app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,33 @@
+class ReviewsController < ApplicationController
+	def index
+	end
+
+	def new
+		@sandwich = Sandwich.find(params[:sandwich_id])
+    @user = current_user
+    @review = @sandwich.reviews.new
+	end
+
+	def create
+		@review = Review.new(review_params)
+    @sandwich = Sandwich.find(params[:sandwich_id])
+    @review.sandwich = @sandwich
+    @user = current_user
+    @review.user = @user
+		if @review.save
+			flash[:notice] = "Review added successfully!"
+			redirect_to root_path
+		else
+			render :new
+		end
+	end
+
+	private
+
+	def review_params
+		params.require(:review).permit(
+			:rating,
+			:body
+		)
+	end
+end
```

Comment:
What, uh, what happened to the indentation, here? Who is responsible for this? **(╯°□°）╯︵ ┻━┻ **

Reply:
It's formatted correctly in Atom.  Any idea what's causing the formatting change or how to address it?